cloudigrade/instance/management/commands/check_instances.py

```diff
@@ -0,0 +1,28 @@
+"""Management command for checking currently-running AWS EC2 instances."""
+import logging
+
+from django.core.management.base import BaseCommand
+from django.utils.translation import ugettext as _
+
+from util import aws
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
```

Comment:
Should this actually be a separate management command? This check is something that runs at account setup, so would it make sense for the `account.management.commands.add_account` to run a function that would do this check instead?

Reply:
Funny timing; I was talking this morning with @adberglund about that same idea but for his work. In this case, I was thinking of our fine-grained approach we had discussed for user stories with step 1 being "do thing independently" and step 2 being "merge disparate things into single command", but after checking I see that this story _doesn't_ have that delineation in its requirements. So, yeah, I think it makes sense to roll it up into the same command.